### PR TITLE
Databricks Reverse ETL source

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Create Databricks Reverse ETL source doc
https://www.notion.so/rudderstacks/Databricks-Reverse-ETL-source-d21860dd489a4015a1eb7949053bfc85

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/reverse-etl/databricks.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+   - **Token** - Enter the personal access token.
+
+<div class="infoBlock">
+
+For more information on getting these settings in Databricks, refer to the <Link to="#faq">FAQ</Link> section below.
+</div>
+
+<div class="successBlock">
+
+  If you've already configured Databricks as a source before, your existing credentials will automatically appear under <strong>Use Existing Credentials</strong>.
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:41]:
```suggestion
If you've already configured Databricks as a source before, your existing credentials will automatically appear under <strong>Use Existing Credentials</strong>.
```

File: docs/reverse-etl/databricks.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   - **Path** - Enter the HTTP path.
+   - **Token** - Enter the personal access token.
+
+<div class="infoBlock">
+
+For more information on getting these settings in Databricks, refer to the <Link to="#faq">FAQ</Link> section below.
+</div>
+
+<div class="successBlock">
+
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:40]:
```suggestion
```

File: docs/reverse-etl/databricks.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+<img src="../assets/warehouse-actions-sources/destination.png" alt="Add destination in RudderStack" />
+
+<div class="infoBlock">
+
+If you have already configured a destination in RudderStack, choose the <strong>Use Existing Destination</strong> option which will take you to the <strong>Schema</strong> tab in the source settings. To add a new destination from scratch, select the <strong>Create New Destination</strong> option which will take you to the destination configuration page.
+</div>
+
+## Specifying the data to import
+
+While connecting a destination to your Databricks source, you can use the default JSON mapping feature.
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:68]: Is VDM not supported for this source? If so, should we specifically call it out?

File: docs/reverse-etl/databricks.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
+---
+# slug: "/docs/reverse-etl/databricks"
+title: "Databricks"
+description: Step-by-step guide to ingest your data from Databricks into RudderStack.
+---
+
+# Databricks
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:7]:
```suggestion
```

Comment (ameypv-rudder) [reply docs/reverse-etl/databricks.mdx:7]: Based on the PR merged yesterday which doesn't require us to add a H1 heading anymore.

File: docs/reverse-etl/databricks.mdx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+## Specifying the data to import
+
+While connecting a destination to your Databricks source, you can use the default JSON mapping feature.
+
+<div class="infoBlock">
+
+For more information on the data import settings, refer to the <a href="https://www.rudderstack.com/docs/reverse-etl/common-settings/importing-data-using-tables/">Importing Data using Tables</a> guide.
+</div>
+
+## FAQ
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:75]: Should we also add a FAQ related to Brian's comment yesterday, that this source doesn't support Mirror Mode or SQL Models?

Comment (ishachopra0401) [reply docs/reverse-etl/databricks.mdx:72]: Not sure.. here's what I think:

Regarding SQL models: The option to choose from either tables/models does not appear in databricks which is quite clear in itself. Also, all the sources which support Models are listed in the Models topic. So, that might be another cue.

Regarding Mirror mode: I think we can mention it in Sync modes topic, like done for other sources.

File: docs/reverse-etl/databricks.mdx (hunk 3 above)
Comment (ishachopra0401) [reply docs/reverse-etl/databricks.mdx:65]: done

File: docs/reverse-etl/databricks.mdx (hunk 5 above)
Comment (ameypv-rudder) [reply docs/reverse-etl/databricks.mdx:76]: Makes sense - let's mention it in the Sync modes doc then!

Comment (ishachopra0401) [reply docs/reverse-etl/databricks.mdx:73]: cool, added in Sync modes! :)

File: docs/reverse-etl/amazon-s3.mdx [hunk 6]
```diff
@@ -17,7 +17,7 @@ To set up Amazon S3 as a source in RudderStack, follow these steps:
 ### Naming the source
 
 1. Log into your [RudderStack dashboard](https://app.rudderstack.com).
-2. From the left panel, select go to **Source** > **New Source** > **Reverse ETL**. Then, select **S3**, as shown:
```
Comment (Vikas26021999) [inline docs/reverse-etl/amazon-s3.mdx:20]: I'm guessing this is probably a typo fix for s3

**Review by Vikas26021999 — APPROVED** [review]
The doc looks great!
LGTM!

File: docs/reverse-etl/amazon-s3.mdx (hunk 6 above)
Comment (ishachopra0401) [reply docs/reverse-etl/amazon-s3.mdx:20]: yep, right!

**Review by ameypv-rudder — APPROVED** [review]

File: docs/reverse-etl/databricks.mdx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+<div class="infoBlock">
+If you have already configured a destination in RudderStack, choose the <strong>Use Existing Destination</strong> option which will take you to the <strong>Schema</strong> tab in the source settings. To add a new destination from scratch, select the <strong>Create New Destination</strong> option which will take you to the destination configuration page.
+</div>
+
+## Specifying the data to import
+
+While connecting a destination to your Databricks source, you can use the default JSON mapping feature.
+
+<div class="infoBlock">
+Note that the <Link to="/reverse-etl/features/visual-data-mapper/">Visual Data Mapping</Link> feature is not supported for Databricks curremtly.
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:66]:
```suggestion
Note that the <Link to="/reverse-etl/features/visual-data-mapper/">Visual Data Mapping</Link> feature is not supported for Databricks currently.
```

File: docs/reverse-etl/databricks.mdx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+<div class="infoBlock">
+For more information on the data import settings, refer to the <a href="https://www.rudderstack.com/docs/reverse-etl/common-settings/importing-data-using-tables/">Importing Data using Tables</a> guide.
+</div>
+
+## FAQ
+
+### Where can I obtain the connection credentials for Databricks?
+
+To obtain the **Host**, **Path**, and **Port** number, follow these steps:
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:77]:
```suggestion
To obtain the **Host**, **Path**, and **Port** number required to set up Databricks source in RudderStack, follow these steps:
```

File: docs/reverse-etl/databricks.mdx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+2. Select the required Databricks resource and click **Launch Workspace**.
+3. Go to the **Compute** tab and select your Databricks cluster.
+4. Click on **Advanced options** > **JDBC/ODBC** tab to find the required settings:
+<img src="../assets/warehouse-actions-sources/connection-settings-databricks.png" alt="Select Databricks source in RudderStack" />
+
+To obtain the **Token**, follow these steps:
+
+1. Log into the [Azure portal](https://portal.azure.com/).
+2. Select the required Databricks resource and click **Launch Workspace**.
+3. Go to **Settings** > **User Settings** to generate personal access token:
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:89]:
```suggestion
3. Go to **Settings** > **User Settings** to generate personal access token, as shown:
```

Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:89]:
```suggestion
3. Go to **Settings** > **User Settings** to generate a new personal access token, as shown:
```

**Comment by ameypv-rudder** [comment]
We're good to go once we resolve the conflict and incorporate the minor suggestions!

**Pull request merged**